src/Raven.Client/Util/JavascriptConversionExtensions.cs

```diff
@@ -2476,32 +2476,23 @@ public override void ConvertToJavascript(JavascriptConversionContext context)
                     {
                         context.Visitor.Visit(innerMemberExpression);
                     }
-                    else
-                    {
-                        writer.Write(alias);
-                    }
 
                     writer.Write(")");
                 }
             }
 
-            internal static bool CanConvert(Expression expression, DocumentConventions conventions, out string aliasName)
-            {
-                return CanConvert(expression, conventions, out aliasName, out _);
-            }
 
-            private static bool CanConvert(Expression expression, DocumentConventions conventions, out string aliasName, out MemberExpression innerMemberExpression)
+            private static bool CanConvert(Expression expression, DocumentConventions conventions, out Expression innerMemberExpression)
```

Comment:
maybe better to change parameter name from `innerMemberExpression` to `innerExpression`, now that you changed it's type to `Expression `